src/main/java/uk/gov/gchq/palisade/service/request/Request.java

```diff
@@ -44,51 +44,48 @@ public Request originalRequestId(final RequestId originalRequestId) {
         return this;
```

Comment:
use setters

Reply:
Ok and I removed the redundant null check